database/gormdb.go

```diff
@@ -385,8 +385,8 @@ func (db *GormDB) UpdateSubmission(sid uint64, approved bool) error {
 	if err := db.conn.First(&submission, sid).Error; err != nil {
 		return err
 	}
-	submission.Approved = approved
-	return db.conn.Model(&pb.Submission{}).Update(submission).Error
+	// submission.Approved = approved
+	return db.conn.Model(&submission).Update("approved", approved).Error
```

Comment:
Don’t understand why pass an “approved” string to the Update?

Reply:
Otherwise the `Approved` field with `false` value would be treated as an empty field and not change. As a result, setting a field to `false`, and passing the whole struct to `Update` would result in the database entry which still has `Approved: true`.